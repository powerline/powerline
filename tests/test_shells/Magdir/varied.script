#------------------------------------------------------------------------------
# $File: varied.script,v 1.10 2014/03/01 22:32:39 christos Exp $
# varied.script:  file(1) magic for various interpreter scripts

0	string/t		#!\ /			a
>3	string		>\0			%s script text executable
!:strength / 2

0	string/b		#!\ /			a
>3	string		>\0			%s script executable (binary data)
!:strength / 2

0	string/t		#!\t/			a
>3	string		>\0			%s script text executable
!:strength / 2

0	string/b		#!\t/			a
>3	string		>\0			%s script executable (binary data)
!:strength / 2

0	string/t		#!/			a
>2	string		>\0			%s script text executable
!:strength / 2

0	string/b		#!/			a
>2	string		>\0			%s script executable (binary data)
!:strength / 2

0	string/t		#!\ 			script text executable
>3	string		>\0			for %s
!:strength / 3

0	string/b		#!\ 			script executable
>3	string		>\0			for %s (binary data)
!:strength / 3

# using env
0	string/t	#!/usr/bin/env		a
>15	string/t	>\0			%s script text executable
!:strength / 10

0	string/b	#!/usr/bin/env		a
>15	string/b	>\0			%s script executable (binary data)
!:strength / 10

0	string/t	#!\ /usr/bin/env	a
>16	string/t	>\0			%s script text executable
!:strength / 10

0	string/b	#!\ /usr/bin/env	a
>16	string/b	>\0			%s script executable (binary data)
!:strength / 10

# From: arno <arenevier@fdn.fr>
# mozilla xpconnect typelib
# see http://www.mozilla.org/scriptable/typelib_file.html
0	string 		XPCOM\nTypeLib\r\n\032		XPConnect Typelib
>0x10  byte        x       version %d
>>0x11 byte        x      \b.%d
