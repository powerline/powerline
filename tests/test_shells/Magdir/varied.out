
#------------------------------------------------------------------------------
# $File: varied.out,v 1.23 2014/04/30 21:41:02 christos Exp $
# varied.out:  file(1) magic for various USG systems
#
#	Herewith many of the object file formats used by USG systems.
#	Most have been moved to files for a particular processor,
#	and deleted if they duplicate other entries.
#
0	short		0610		Perkin-Elmer executable
# AMD 29K
0	beshort		0572		amd 29k coff noprebar executable
0	beshort		01572		amd 29k coff prebar executable
0	beshort		0160007		amd 29k coff archive
# Cray
6	beshort		0407		unicos (cray) executable
# Ultrix 4.3
596	string		\130\337\377\377	Ultrix core file
>600	string		>\0		from '%s'
# BeOS and MAcOS PEF executables
# From: hplus@zilker.net (Jon Watte)
0	string		Joy!peffpwpc	header for PowerPC PEF executable
#
# ava assembler/linker Uros Platise <uros.platise@ijs.si>
0       string          avaobj  AVR assembler object code
>7      string          >\0     version '%s'
# gnu gmon magic From: Eugen Dedu <dedu@ese-metz.fr>
0	string		gmon		GNU prof performance data
>4	long		x		- version %d
# From: Dave Pearson <davep@davep.org>
# Harbour <URL:http://harbour-project.org/> HRB files.
0	string		\xc0HRB		Harbour HRB file
>4	leshort		x		version %d
# Harbour HBV files
0	string		\xc0HBV		Harbour variable dump file
>4	leshort		x		version %d

# From: Alex Beregszaszi <alex@fsn.hu>
# 0	string		exec 		BugOS executable
# 0	string		pack		BugOS archive

# From: Jason Spence <jspence@lightconsulting.com>
# Generated by the "examples" in STM's ST40 devkit, and derived code.
0	lelong		0x13a9f17e	ST40 component image format
>4	string		>\0		\b, name '%s'

